src/components/Sidebar/Sidebar.jsx

```diff
@@ -34,8 +34,12 @@ class Sidebar extends React.Component {
     this.handleLogoutClick = this.handleLogoutClick.bind(this);
   }
 
+  componentDidMount(){
+    this.state.activeState.dashboard === true && this.props.history.push(`/dashboard`);
```

Comment:
is there a reason we doing === true

Reply:
because if I click on profile, this.state.activeState.profile is true. When I refresh this.state.activeState.dashboard is true. but I am still on the Profile page. It has to show the correct page based on the active sidebar state.

Reply:
This did not answer @dannylekim question. He's just asking why you write `true`

just this is ok `this.state.activeState.dashboard && this.props.history.push(`/dashboard`);`